### ISSUE TITLE
Home App Automation

### ISSUE THREAD

**Issue by lenkai** [nfarina/homebridge#1444]
Hey,

i have the folling homebridge plugins installed:

https://www.npmjs.com/package/homebridge-advanced-http-temperature-humidity
https://www.npmjs.com/package/homebridge-http-ambient-light

and in the Home App on my iPad, i want to create a rule/automation (i.e. lightlevel under 2000 Lux -> turn on lights etc.), but if i want to create this automation, the option "sensor detects something" (in the app) is greyed out.

Thanks for your advices
Lennart

**Comment by maragelis** [comment]
That automation will be available in ios 11 in ios10 you cannot use sensors that output values to create an automation, it is grayed out because you do not have any sensors that output true false. 
There is a workaround, use the EVE app to create automation`s this app allows for sensor value condition triggering.

**Comment by viictor924** [comment]
Hmm, have either of you guys had luck setting this up now that iOS 11 is released? 

I've got a temperature and humidity sensor connected to homekit but the "sensor detects something" automation is still grayed out. I was able to use the EVE app workaround but it would be nice if it was all done via the Home app. 

![img_0075 jpg](https://user-images.githubusercontent.com/23468631/31376990-810e19b0-ad5b-11e7-888f-e7b44c32759e.JPG)

![img_0074 jpg](https://user-images.githubusercontent.com/23468631/31376996-862e991a-ad5b-11e7-8af9-196e9a11244b.JPG)
